src/models/credit.js

```diff
@@ -0,0 +1,11 @@
+import mongoose from "mongoose";
+
+import database from "../database.js";
+
+const creditSchema = new mongoose.Schema({
+
+    amount: Number
+
+});
```

Comment:
This schema is too basic, it should include the status for the transaction (OK, ERROR, TIMEOUT)

Reply:
the statuses are  for the message Model. there is only one Entry in the credit database, and it has no status, its shows only the current balance.